Clarified parser behavior in documentation (#1392)
fluent/fluent-bit-docs: pipeline/inputs/tail.md

{% endtab %}
{% endtabs %}

> The two options separated by a comma mean Fluent Bit will try each parser in the list in order, applying the first one that matches the log (i.e. use the first parser which has a `start_state` that matches the log). For example, it will first try `docker`, and if `docker` does not match, it will then try `cri`.

Reviewer: Our style would have us wrap lines around 80 characters in length, and we avoid words like i.e.

Does this suggestion still technically describe the behavior of Fluent Bit here?

Mentioning @alexakreizinger for awareness.

```suggestion
Having the the two options separated by a comma causes Fluent Bit to attempt each
parser in the list in order. Fluent But will apply the first parser that matches the
log (such as the first parser that has a `start_state` that matches the log). For
example, Fluent Bit will first try `docker`, and if `docker` doesn't match, it will
then try `cri`.
```
